tools/sgx-lkl-docker

```diff
@@ -23,17 +23,16 @@ function usage() {
     echo "`basename $0` <command>"
     echo
     echo "where <command> is one of the following:"
-    echo "  build-cc                                           Build confidential container" 
+    echo "  build                                              Re-package disk images and configuration as Docker image" 
```

Comment:
I would still say:

```suggestion
    echo "  build                                              Build confidential container from disk images and configuration files" 
```

Reply:
`docker build` is technically unable to build containers. It builds images. And hence we can't use that term in this context. This has always bothered me and I find it quite confusing.